test/vagrant-spec/Vagrantfile.spec

```diff
@@ -83,36 +91,67 @@ Vagrant.configure(2) do |global_config|
           config.vm.provision :shell,
             path: "./scripts/#{PLATFORM_SCRIPT_MAPPING[platform]}-setup.#{provider_name}.sh", run: "once"
         end
-        guest_boxes.each_with_index do |box_info, idx|
-          guest_box, box_version = box_info
-          guest_platform = guest_box.split('/').last.sub(/[^a-z]+$/, '')
-          guest_platform = PLATFORM_SCRIPT_MAPPING[guest_platform]
-          spec_cmd_args = ENV["VAGRANT_SPEC_ARGS"]
-          if idx != 0
-            spec_cmd_args = "#{spec_cmd_args} --without-component cli/*".strip
+        if provider_name == "docker"
+          docker_images.each_with_index do |image_info, idx|
+            docker_image, _ = image_info
+            spec_cmd_args = ENV["VAGRANT_SPEC_ARGS"]
+            if idx != 0
+              spec_cmd_args = "#{spec_cmd_args} --without-component cli/*".strip
+            end
+            if platform == "windows"
+              config.vm.provision(
+                :shell,
+                path: "./scripts/#{platform}-run.#{provider_name}.ps1",
+                keep_color: true,
+                env: {
+                  "VAGRANT_SPEC_ARGS" => "--no-builtin --component provider/docker/docker/* #{spec_cmd_args}".strip,
```

Comment:
I don't know if I like this. But the idea here is, the vagrant-spec builtin tests don't really work for the docker provider (since they are driven by boxes which are not applicable to the docker provider). So, only run docker specific provider tests.

Reply:
Chatted about this and decided that this is fine for now to isolate the specific tests. Once we have a proper box included within the test matrix we can pull this bit out.